# APP STORE ANALYSIS (GOOGLE and APPLE)

## A team of developers want to develop an app for both google and apple.
## They are intending to publish a free app and thinking to generate revenue solely on ads.
## This will require to have large number of users for our application.

## Final goal is to to come up with category of apps that would attract more users by analyzing below datasets.

- Data is taken from kaggle website.
- Google play store data and Apple ios app store data
- [Link](https://dq-content.s3.amazonaws.com/350/googleplaystore.csv) GooglePlayStoreData
  [Link](https://www.kaggle.com/lava18/google-play-store-apps) GooglePlayStoreApps
- [Link](https://dq-content.s3.amazonaws.com/350/AppleStore.csv) AppleiosStoreData
  [Link](https://www.kaggle.com/ramamet4/app-store-apple-data-set-10k-apps/home) MobileAppStore

In [1]:
openfile = open("AppleStore.csv",encoding="utf8")
from csv import reader
readfile = reader(openfile)
ios = list(readfile)
ios_header = ios[0]
ios = ios[1:]

openfile = open("googleplaystore.csv",encoding="utf8")
from csv import reader
readfile = reader(openfile)
android = list(readfile)
android_header = android[0]
android = android[1:]

In [2]:
def explore_dataset(dataset,start,end,rows_columns=False):
    dataset_slice = dataset[start:end]
    for row in dataset_slice:
        print(row)
        print('\n')
        
    if rows_columns:
        print("Number of rows = ", len(dataset))
        print("Number of columns = ", len(dataset[0]))

In [3]:
android_header

['App',
 'Category',
 'Rating',
 'Reviews',
 'Size',
 'Installs',
 'Type',
 'Price',
 'Content Rating',
 'Genres',
 'Last Updated',
 'Current Ver',
 'Android Ver']

- App,category,rating,reviews,installs,type,price,Gneres are useful for our analysis
- Therr are 10,841 rows in android dataset and 13 columns.

In [4]:
explore_dataset(android,0,1,rows_columns=True)


['Photo Editor & Candy Camera & Grid & ScrapBook', 'ART_AND_DESIGN', '4.1', '159', '19M', '10,000+', 'Free', '0', 'Everyone', 'Art & Design', 'January 7, 2018', '1.0.0', '4.0.3 and up']


Number of rows =  10841
Number of columns =  13


In [5]:
ios_header

['id',
 'track_name',
 'size_bytes',
 'currency',
 'price',
 'rating_count_tot',
 'rating_count_ver',
 'user_rating',
 'user_rating_ver',
 'ver',
 'cont_rating',
 'prime_genre',
 'sup_devices.num',
 'ipadSc_urls.num',
 'lang.num',
 'vpp_lic']

- id,trackname,price,rating_count_tot,rating_count_ver,user_rating,user_rating_ver,cont_rating,prime_genre are useful for our analysis
- There are 7197 rows and 16 columns in ios appstore

In [6]:
explore_dataset(ios,0,1,rows_columns=True)

['284882215', 'Facebook', '389879808', 'USD', '0.0', '2974676', '212', '3.5', '3.5', '95.0', '4+', 'Social Networking', '37', '1', '29', '1']


Number of rows =  7197
Number of columns =  16


[Link](https://www.kaggle.com/ramamet4/app-store-apple-data-set-10k-apps/home?select=appleStore_description.csv)

- You can find the description of ios columns in the above link

In [11]:
header_length = len(android_header)
for row in android:
    row_length = len(row)
    if header_length != row_length:
        print(row)
        print('\n')
        print(android.index(row))

['Life Made WI-Fi Touchscreen Photo Frame', '1.9', '19', '3.0M', '1,000+', 'Free', '0', 'Everyone', '', 'February 11, 2018', '1.0.19', '4.0 and up']


10472


In [12]:
android[10472]

['Life Made WI-Fi Touchscreen Photo Frame',
 '1.9',
 '19',
 '3.0M',
 '1,000+',
 'Free',
 '0',
 'Everyone',
 '',
 'February 11, 2018',
 '1.0.19',
 '4.0 and up']

In [13]:
del android[10472]

In [18]:
unique_apps=[]
duplicate_apps=[]

for row in android:
        app_name = row[0]
        if app_name in unique_apps:
            duplicate_apps.append(app_name)
        else:
            unique_apps.append(app_name)
print("Number of duplicate apps are",len(duplicate_apps))
# print("Duplicate Apps are", duplicate_apps)

Number of duplicate apps are 1181


In [20]:
unique_apps=[]
duplicate_apps=[]

for row in ios:
        app_name = row[1]
        if app_name in unique_apps:
            duplicate_apps.append(app_name)
        else:
            unique_apps.append(app_name)
print("Number of duplicate apps are",len(duplicate_apps))
print("Duplicate Apps are", duplicate_apps)

Number of duplicate apps are 2
Duplicate Apps are ['Mannequin Challenge', 'VR Roller Coaster']


- As we can see from above cells there are 1181 duplicates in android and 2 in ios which we need to analyse and confirm if they are duplicates really.

In [28]:
for row in android:
    if row[0] == 'Instagram':
        print(row)

['Instagram', 'SOCIAL', '4.5', '66577313', 'Varies with device', '1,000,000,000+', 'Free', '0', 'Teen', 'Social', 'July 31, 2018', 'Varies with device', 'Varies with device']
['Instagram', 'SOCIAL', '4.5', '66577446', 'Varies with device', '1,000,000,000+', 'Free', '0', 'Teen', 'Social', 'July 31, 2018', 'Varies with device', 'Varies with device']
['Instagram', 'SOCIAL', '4.5', '66577313', 'Varies with device', '1,000,000,000+', 'Free', '0', 'Teen', 'Social', 'July 31, 2018', 'Varies with device', 'Varies with device']
['Instagram', 'SOCIAL', '4.5', '66509917', 'Varies with device', '1,000,000,000+', 'Free', '0', 'Teen', 'Social', 'July 31, 2018', 'Varies with device', 'Varies with device']


- As we can see from instagram data the duplicate rows varies by number of reviews.
- More the reviews latest would be the data.
- we will keep the rows with highest reviews and remove the other apps from the dataset

In [30]:
for row in ios:
    if row[1] == 'Mannequin Challenge':
        print(row)
        
for row in ios:
    if row[1] == 'VR Roller Coaster':
        print(row)

['1173990889', 'Mannequin Challenge', '109705216', 'USD', '0.0', '668', '87', '3.0', '3.0', '1.4', '9+', 'Games', '37', '4', '1', '1']
['1178454060', 'Mannequin Challenge', '59572224', 'USD', '0.0', '105', '58', '4.0', '4.5', '1.0.1', '4+', 'Games', '38', '5', '1', '1']
['952877179', 'VR Roller Coaster', '169523200', 'USD', '0.0', '107', '102', '3.5', '3.5', '2.0.0', '4+', 'Games', '37', '5', '1', '1']
['1089824278', 'VR Roller Coaster', '240964608', 'USD', '0.0', '67', '44', '3.5', '4.0', '0.81', '4+', 'Games', '38', '0', '1', '1']


- many values are differeing for ios data , more importantly versions and they might also keep older versions while creating newer ones in the app store.
- so we are not considering them as duplicate.

In [32]:
reviews_max = {}

for row in android:
    app = row[0]
    n_reviews = float(row[3])
    if app in reviews_max and reviews_max[app] < n_reviews:
        reviews_max[app] = n_reviews
    if app not in reviews_max:
        reviews_max[app] = n_reviews
        
print(len(reviews_max))

9659


In [33]:
android_clean = []
already_added = []

for row in android:
    app = row[0]
    n_reviews = float(row[3])
    if n_reviews == reviews_max[app] and app not in already_added:
        android_clean.append(row)
        already_added.append(app)
        
print(len(android_clean))

9659


- reviews_max dictionary is created to store apps with highest reviews
- android_clean list of lists is created by selecting apps with highest reviews (using reviews_max dictionary)

In [49]:
def check_if_english_name(app_name):
    count = 0
    for character in app_name:
        if ord(character)>127:
            count += 1
    if count>3:
        return False
    else:
        return True

print(check_if_english_name('Instagram'))
print(check_if_english_name('爱奇艺PPS -《欢乐颂2》电视剧热播'))
print(check_if_english_name('Docs To Go™ Free Office Suite'))
print(check_if_english_name('Instachat 😜'))

True
False
True
True


In [51]:
android_english =[]
ios_english = []

for row in android_clean:
    if check_if_english_name(row[0]):
        android_english.append(row)
        
for row in ios:
    if check_if_english_name(row[1]):
        ios_english.append(row)
        
        
print(len(android_english))
print(len(ios_english))

9614
6183


In [53]:
android_final=[]
ios_final=[]

for row in android_english:
    if row[7] == '0':
        android_final.append(row)
        
for row in ios_english:
    if row[4] == '0.0':
        ios_final.append(row)

explore_dataset(android_final,0,1,rows_columns=True)
explore_dataset(ios_final,0,1,rows_columns=True)

['Photo Editor & Candy Camera & Grid & ScrapBook', 'ART_AND_DESIGN', '4.1', '159', '19M', '10,000+', 'Free', '0', 'Everyone', 'Art & Design', 'January 7, 2018', '1.0.0', '4.0.3 and up']


Number of rows =  8864
Number of columns =  13
['284882215', 'Facebook', '389879808', 'USD', '0.0', '2974676', '212', '3.5', '3.5', '95.0', '4+', 'Social Networking', '37', '1', '29', '1']


Number of rows =  3222
Number of columns =  16


- Removed inaccurate data
- Removed duplicate data
- Removed non-english apps
- Filtered Free apps

- Build a minimal version of android app and release in google play store
- If the app gets a good response, develop it further
- If the app is profitable after 6 months, release on ios store

- As our end goal is to release the app in both android and ios playstores, we need to find app profiles that are successful in both markets
- For this we will use Genres column in android dataset and primeGenre column in ios dataset

In [95]:

def freq_table(dataset,index):
    freq_dict = {}
    freq_dict_per = {}
    total = 0
    for row in dataset:
        total += 1
        key = row[index]
        if key in freq_dict:
            freq_dict[key] += 1
        else:
            freq_dict[key] = 1

    for key in freq_dict:
        freq_dict_per[key] = (freq_dict[key]/total)*100
    return freq_dict_per

In [101]:
def display_table(dataset,index):
    table = freq_table(dataset,index)
    table_display = []
    for key in table:
        key_value_as_tuple = (table[key],key)
        table_display.append(key_value_as_tuple)
    sorted_table_display = sorted(table_display,reverse=True)

    for element in sorted_table_display:
        print(element[1],":",element[0])

In [102]:
display_table(android_final,1)

FAMILY : 18.907942238267147
GAME : 9.724729241877256
TOOLS : 8.461191335740072
BUSINESS : 4.591606498194946
LIFESTYLE : 3.9034296028880866
PRODUCTIVITY : 3.892148014440433
FINANCE : 3.7003610108303246
MEDICAL : 3.531137184115524
SPORTS : 3.395758122743682
PERSONALIZATION : 3.3167870036101084
COMMUNICATION : 3.2378158844765346
HEALTH_AND_FITNESS : 3.0798736462093865
PHOTOGRAPHY : 2.944494584837545
NEWS_AND_MAGAZINES : 2.7978339350180503
SOCIAL : 2.6624548736462095
TRAVEL_AND_LOCAL : 2.33528880866426
SHOPPING : 2.2450361010830324
BOOKS_AND_REFERENCE : 2.1435018050541514
DATING : 1.861462093862816
VIDEO_PLAYERS : 1.7937725631768955
MAPS_AND_NAVIGATION : 1.3989169675090252
FOOD_AND_DRINK : 1.2409747292418771
EDUCATION : 1.1620036101083033
ENTERTAINMENT : 0.9589350180505415
LIBRARIES_AND_DEMO : 0.9363718411552346
AUTO_AND_VEHICLES : 0.9250902527075812
HOUSE_AND_HOME : 0.8235559566787004
WEATHER : 0.8009927797833934
EVENTS : 0.7107400722021661
PARENTING : 0.6543321299638989
ART_AND_DESIGN : 

- Family category of apps which are games for kids are a little higher in number compared to other categories.
- Overall we can see that google play store has good and almost equal contribution of both entertainment and practical purpose apps.

In [104]:
display_table(android_final,9)

Tools : 8.449909747292418
Entertainment : 6.069494584837545
Education : 5.347472924187725
Business : 4.591606498194946
Productivity : 3.892148014440433
Lifestyle : 3.892148014440433
Finance : 3.7003610108303246
Medical : 3.531137184115524
Sports : 3.463447653429603
Personalization : 3.3167870036101084
Communication : 3.2378158844765346
Action : 3.1024368231046933
Health & Fitness : 3.0798736462093865
Photography : 2.944494584837545
News & Magazines : 2.7978339350180503
Social : 2.6624548736462095
Travel & Local : 2.3240072202166067
Shopping : 2.2450361010830324
Books & Reference : 2.1435018050541514
Simulation : 2.0419675090252705
Dating : 1.861462093862816
Arcade : 1.8501805054151623
Video Players & Editors : 1.7712093862815883
Casual : 1.7599277978339352
Maps & Navigation : 1.3989169675090252
Food & Drink : 1.2409747292418771
Puzzle : 1.128158844765343
Racing : 0.9927797833935018
Role Playing : 0.9363718411552346
Libraries & Demo : 0.9363718411552346
Auto & Vehicles : 0.9250902527075

- As more specific information is included in generes column we will go with analyzing category column to get overall picture,

In [103]:
display_table(ios_final,-5)

Games : 58.16263190564867
Entertainment : 7.883302296710118
Photo & Video : 4.9658597144630665
Education : 3.662321539416512
Social Networking : 3.2898820608317814
Shopping : 2.60707635009311
Utilities : 2.5139664804469275
Sports : 2.1415270018621975
Music : 2.0484171322160147
Health & Fitness : 2.0173805090006205
Productivity : 1.7380509000620732
Lifestyle : 1.5828677839851024
News : 1.3345747982619491
Travel : 1.2414649286157666
Finance : 1.1173184357541899
Weather : 0.8690254500310366
Food & Drink : 0.8069522036002483
Reference : 0.5586592178770949
Business : 0.5276225946617008
Book : 0.4345127250155183
Navigation : 0.186219739292365
Medical : 0.186219739292365
Catalogs : 0.12414649286157665


- Apple store is dominated by games and entertainment.
- Apple store has apps that are fun intended.

- Having high percentage of certain category or genre doesnt signify high number of users.
- Hence we need to find the category or genre with high users

- we can find number of users per category using installs in google play  store.
- since installs is not available in apple store, we can use average rating_count_tot per genre

In [105]:
prime_genre_freq_table = freq_table(ios_final,-5)
prime_genre_freq_table

{'Social Networking': 3.2898820608317814,
 'Photo & Video': 4.9658597144630665,
 'Games': 58.16263190564867,
 'Music': 2.0484171322160147,
 'Reference': 0.5586592178770949,
 'Health & Fitness': 2.0173805090006205,
 'Weather': 0.8690254500310366,
 'Utilities': 2.5139664804469275,
 'Travel': 1.2414649286157666,
 'Shopping': 2.60707635009311,
 'News': 1.3345747982619491,
 'Navigation': 0.186219739292365,
 'Lifestyle': 1.5828677839851024,
 'Entertainment': 7.883302296710118,
 'Food & Drink': 0.8069522036002483,
 'Sports': 2.1415270018621975,
 'Book': 0.4345127250155183,
 'Finance': 1.1173184357541899,
 'Education': 3.662321539416512,
 'Productivity': 1.7380509000620732,
 'Business': 0.5276225946617008,
 'Catalogs': 0.12414649286157665,
 'Medical': 0.186219739292365}

In [112]:
avg_tot_ratings_per_genre = {}
for key in prime_genre_freq_table:
    total_num_ratings_per_genre = 0
    apps_per_genre = 0
    for row in ios_final:
        if row[-5] == key:
            total_num_ratings_per_genre += float(row[5])
            apps_per_genre += 1
    avg_tot_ratings_per_genre[key] = total_num_ratings_per_genre/apps_per_genre

def display_users(dictionary):   
    table_display = []
    for key in dictionary:
        key_value_as_tuple = (dictionary[key],key)
        table_display.append(key_value_as_tuple)
    sorted_table_display = sorted(table_display,reverse=True)

    for element in sorted_table_display:
        print(element[1],":",element[0])
            
display_users(avg_tot_ratings_per_genre)

Navigation : 86090.33333333333
Reference : 74942.11111111111
Social Networking : 71548.34905660378
Music : 57326.530303030304
Weather : 52279.892857142855
Book : 39758.5
Food & Drink : 33333.92307692308
Finance : 31467.944444444445
Photo & Video : 28441.54375
Travel : 28243.8
Shopping : 26919.690476190477
Health & Fitness : 23298.015384615384
Sports : 23008.898550724636
Games : 22788.6696905016
News : 21248.023255813954
Productivity : 21028.410714285714
Utilities : 18684.456790123455
Lifestyle : 16485.764705882353
Entertainment : 14029.830708661417
Business : 7491.117647058823
Education : 7003.983050847458
Catalogs : 4004.0
Medical : 612.0


- we find that navigation, reference,social networking,music,weather,book,food and drink and finance have more total user ratings.
- Lets explore them individually to know further

In [122]:
explore_list_ios = ["Navigation","Reference","Social Networking","Music","Weather","Book","Food & Drink","Finance"]

for prime_genre in explore_list_ios:
    print(prime_genre)
    print("\n")
    for row in ios_final:
        if row[-5] == prime_genre:
            print(row[1],":",row[5])
    print("\n")
           
           

Navigation


Waze - GPS Navigation, Maps & Real-time Traffic : 345046
Google Maps - Navigation & Transit : 154911
Geocaching® : 12811
CoPilot GPS – Car Navigation & Offline Maps : 3582
ImmobilienScout24: Real Estate Search in Germany : 187
Railway Route Search : 5


Reference


Bible : 985920
Dictionary.com Dictionary & Thesaurus : 200047
Dictionary.com Dictionary & Thesaurus for iPad : 54175
Google Translate : 26786
Muslim Pro: Ramadan 2017 Prayer Times, Azan, Quran : 18418
New Furniture Mods - Pocket Wiki & Game Tools for Minecraft PC Edition : 17588
Merriam-Webster Dictionary : 16849
Night Sky : 12122
City Maps for Minecraft PE - The Best Maps for Minecraft Pocket Edition (MCPE) : 8535
LUCKY BLOCK MOD ™ for Minecraft PC Edition - The Best Pocket Wiki & Mods Installer Tools : 4693
GUNS MODS for Minecraft PC Edition - Mods Tools : 1497
Guides for Pokémon GO - Pokemon GO News and Cheats : 826
WWDC : 762
Horror Maps for Minecraft PE - Download The Scariest Maps for Minecraft Pocket Edit

- We can see from above results that popular genres like navigation,social networking,music are dominated by famous apps like (waze,google maps), (facebook,pininterest),(pandora,spotify) respectively.
- This segment is saturated already. As it is evident that apple store is highly dominated by fun segment.
# Reference section seems some potential for new entries as it is only dominated by Bible and there is a scope for new entries.


- Therefore we can explore books and dictionaries section for apple store. This overlaps with "book" genre.
- Rest Finance, food and drink, weather doesn't have much to offer.

In [123]:
category_freq_table = freq_table(android_final,1)
category_freq_table

{'ART_AND_DESIGN': 0.6430505415162455,
 'AUTO_AND_VEHICLES': 0.9250902527075812,
 'BEAUTY': 0.5979241877256317,
 'BOOKS_AND_REFERENCE': 2.1435018050541514,
 'BUSINESS': 4.591606498194946,
 'COMICS': 0.6204873646209386,
 'COMMUNICATION': 3.2378158844765346,
 'DATING': 1.861462093862816,
 'EDUCATION': 1.1620036101083033,
 'ENTERTAINMENT': 0.9589350180505415,
 'EVENTS': 0.7107400722021661,
 'FINANCE': 3.7003610108303246,
 'FOOD_AND_DRINK': 1.2409747292418771,
 'HEALTH_AND_FITNESS': 3.0798736462093865,
 'HOUSE_AND_HOME': 0.8235559566787004,
 'LIBRARIES_AND_DEMO': 0.9363718411552346,
 'LIFESTYLE': 3.9034296028880866,
 'GAME': 9.724729241877256,
 'FAMILY': 18.907942238267147,
 'MEDICAL': 3.531137184115524,
 'SOCIAL': 2.6624548736462095,
 'SHOPPING': 2.2450361010830324,
 'PHOTOGRAPHY': 2.944494584837545,
 'SPORTS': 3.395758122743682,
 'TRAVEL_AND_LOCAL': 2.33528880866426,
 'TOOLS': 8.461191335740072,
 'PERSONALIZATION': 3.3167870036101084,
 'PRODUCTIVITY': 3.892148014440433,
 'PARENTING': 0.6

# Lets explore Google Play Store users using installs column


- Since this column contains "+" and "," characters, we need to replace them with spaces and perform numerical operations after converting them to float type.

In [127]:
average_installs_per_category = {}
for key in category_freq_table:
    total_installs = 0
    apps_per_category=0
    for row in android_final:
        if row[1] == key:
            installs = row[5]
            installs = installs.replace("+","")
            installs = installs.replace(",","")
            installs = float(installs)
            total_installs += installs
            apps_per_category += 1
    average_installs_per_category[key] = total_installs/apps_per_category
    
display_users(average_installs_per_category)

COMMUNICATION : 38456119.167247385
VIDEO_PLAYERS : 24727872.452830188
SOCIAL : 23253652.127118643
PHOTOGRAPHY : 17840110.40229885
PRODUCTIVITY : 16787331.344927534
GAME : 15588015.603248259
TRAVEL_AND_LOCAL : 13984077.710144928
ENTERTAINMENT : 11640705.88235294
TOOLS : 10801391.298666667
NEWS_AND_MAGAZINES : 9549178.467741935
BOOKS_AND_REFERENCE : 8767811.894736841
SHOPPING : 7036877.311557789
PERSONALIZATION : 5201482.6122448975
WEATHER : 5074486.197183099
HEALTH_AND_FITNESS : 4188821.9853479853
MAPS_AND_NAVIGATION : 4056941.7741935486
FAMILY : 3695641.8198090694
SPORTS : 3638640.1428571427
ART_AND_DESIGN : 1986335.0877192982
FOOD_AND_DRINK : 1924897.7363636363
EDUCATION : 1833495.145631068
BUSINESS : 1712290.1474201474
LIFESTYLE : 1437816.2687861272
FINANCE : 1387692.475609756
HOUSE_AND_HOME : 1331540.5616438356
DATING : 854028.8303030303
COMICS : 817657.2727272727
AUTO_AND_VEHICLES : 647317.8170731707
LIBRARIES_AND_DEMO : 638503.734939759
PARENTING : 542603.6206896552
BEAUTY : 51315

In [128]:
explore_list_android = ["COMMUNICATION","VIDEO_PLAYERS","SOCIAL","PHOTOGRAPHY","PRODUCTIVITY","GAME","TRAVEL_AND_LOCAL","ENTERTAINMENT"]

for category in explore_list_android:
    print(category)
    print("\n")
    for row in android_final:
        if row[1] == category:
            print(row[0],":",row[5])
    print("\n")
           
           

COMMUNICATION


WhatsApp Messenger : 1,000,000,000+
Messenger for SMS : 10,000,000+
My Tele2 : 5,000,000+
imo beta free calls and text : 100,000,000+
Contacts : 50,000,000+
Call Free – Free Call : 5,000,000+
Web Browser & Explorer : 5,000,000+
Browser 4G : 10,000,000+
MegaFon Dashboard : 10,000,000+
ZenUI Dialer & Contacts : 10,000,000+
Cricket Visual Voicemail : 10,000,000+
TracFone My Account : 1,000,000+
Xperia Link™ : 10,000,000+
TouchPal Keyboard - Fun Emoji & Android Keyboard : 10,000,000+
Skype Lite - Free Video Call & Chat : 5,000,000+
My magenta : 1,000,000+
Android Messages : 100,000,000+
Google Duo - High Quality Video Calls : 500,000,000+
Seznam.cz : 1,000,000+
Antillean Gold Telegram (original version) : 100,000+
AT&T Visual Voicemail : 10,000,000+
GMX Mail : 10,000,000+
Omlet Chat : 10,000,000+
My Vodacom SA : 5,000,000+
Microsoft Edge : 5,000,000+
Messenger – Text and Video Chat for Free : 1,000,000,000+
imo free video calls and chat : 500,000,000+
Calls & Text by Mo+ : 

Turbo AI™ - Personal assistant : 5,000+
AI Sight : 1,000+
AL Voice Recorder : 1,000,000+
AP ENPS Mobile : 5,000+
AQ Guards : 10+
QR Droid : 50,000,000+
Mind Map AR, Augmented Reality ARCore Mind Mapping : 5,000+
PDF Viewer & Book Reader : 5,000,000+
Alarm Clock: Stopwatch & Timer : 50,000,000+
Total AV : 100,000+
Mobile Security: Antivirus, Web Scan & App Lock : 10,000,000+
Professional AV Solution & Products information : 1,000+
Bits Watch Face : 100,000+
Tips Microsoft Dynamics Ax : 500+
Tips & Tricks Dynamics AX 365 : 100+
A-Z App Store : 500,000+
BA Calculator : 5,000+
Town of Princeton, BC : 100+
Electricity Bill Calculator BD : 10,000+
BD Attendance : 1,000+
BD Field Force : 1,000+
BG+Phone Backup : 1,000+
ALL-IN-ONE PACKAGE TRACKING : 1,000,000+
MinT BI : 10+
BJ Memo Widget : 50,000+
SKIN BK : 10+
BM Interventi : 10+
BAMMS for BM 1Park : 50+
BAMMS for BM SQ : 10+
BN Inscript Improved Input keyboard : 10+
Calendars (En, Bn, Ar) : 1,000+
BN DB1 App : 50+
BN Inscript Improved Keybo

search.ch : 500,000+
CJ's Coffee Cafe : 500+
Camping and Co - Camping in Europe : 1,000+
CQ Hotels : 10,000+
SHUTTLLS CQ - Connect Ride Go : 5+
Cathay Pacific : 1,000,000+
Map of Cyprus offline : 10,000+
DB BAHN : 500+
DB FahrtProfi : 1,000+
DC Metro Transit - Free : 500,000+
DC Trails-Hop On Hop Off Tours : 5,000+
Can I pack that? - DG App : 10,000+
DG Cars : 10,000+
DK Eyewitness Audio Walks : 1,000+
DM Transfers Dalaman Transfers : 10,000+
US DV Lottery 2020 Apply : 10,000+
US DV Lottery 2019 : 1,000+
DW Maps : 10,000+
junainfo.dy.fi : 100+
Siyaha Dz : 10,000+
IRIS : Customer Service - DZ Algeria : 1,000+
DZ Fly Algérie Horaire Vols : 10,000+
EA Plus : 1,000+
EB Events : 10,000+
Audiowalk EB : 10+
EF Go Ahead Tour Companion : 1,000+
EF Summits : 100+
EG SIM CARD (EGSIMCARD) : 10,000+
The Emirates App : 1,000,000+
Ek-yatri: Travel where you Belong : 1+
EL AL Flights : 100,000+
DreamStream By EL AL : 100,000+
El Dorado App : 10,000+
EO Hotels : 100+
Fill 'er Up : 5,000+
MAPS.ME – Offl

- It seems that google play store also follows the same pattern where the categories are dominated by popular apps like whatsapp, facebook,youtube etc.
- Few famous apps are capturing most of the segment.
- Since we analyzed books would be a good niche for apple store we will try to analyze "BOOKS_AND_REFERENCE" section in Google play store as well

In [131]:
books_ref = {}
for row in android_final:
        if row[1] =="BOOKS_AND_REFERENCE" :
            books_ref[row[0]] = row[5]
display_users( books_ref)

Youboox - Livres, BD et magazines : 500,000+
SDA Sabbath School Quarterly : 500,000+
Recipes of Prophetic Medicine for free : 500,000+
Only 30 days in English, the guideline is guaranteed : 500,000+
Offline: English to Tagalog Dictionary : 500,000+
NOOK Audiobooks : 500,000+
NOOK App for NOOK Devices : 500,000+
Google I/O 2018 : 500,000+
Golden Dictionary (FR-AR) : 500,000+
Flybook : 500,000+
English to Urdu Dictionary : 500,000+
English Persian Dictionary : 500,000+
English Grammar Complete Handbook : 500,000+
Azpen eReader : 500,000+
Al-Quran 30 Juz free copies : 500,000+
Al Quran (Tafsir & by Word) : 500,000+
SDA Collegiate Quarterly : 500+
Modlitební knížka CZ : 500+
Learn SAP BW on HANA : 500+
Learn SAP BW : 500+
Le Fe de Jesus : 500+
La Fe de Jesús : 500+
Catholic La Bu Zo Kam : 500+
eBoox new: Reader for fb2 epub zip books : 50,000+
SH-02J Owner's Manual (Android 8.0) : 50,000+
La citadelle du musulman : 50,000+
Fertilizer Removal By Crop : 50,000+
EZ Quran : 50,000+
E-Book Read

- Even this segment is flooded by variety of apps covering various aspects.
- Since this segment is not dominated by any famous app like that of facebook, wahtsapp or youtube, this category has some potential for entry.
- If we can come up with any brilliant ideas which are not covered in the above listed apps and also with some marketing tactics  we would be able to make it popular. 




# Conclusion:

## Books was one genre where we showed our interest in Apple store as well, this category would be worthy enough to explore as it doesn't contain any market leader and brilliant ideas will definitely get our work done.